app/components/Welcome/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
Also I recommend for all routes to place them in a ```routes``` folder. Following the next schema:
```
src/routes/welcome
    + components
           + Layout.tsx
           + MyPrivateComponent
                    +index.jsx
                    + index.scss
       + store
                + (actions | selectors | reducers | models) / If you need them
        index.jsx //React container
        selector.js //Selectors used in the container done with reselect
        actions.js //Basically the mapToDispatch object  used in the container
```

In that way you have isolated the specific code of routes, easier to maintain, easier to debug.

Reply:
Of course, that is a "complex" example, if in your route you do not need anything from the store, you can skip the **store** folder (etc...)